stable/spark/templates/spark-master-deployment.yaml

```diff
@@ -18,6 +18,10 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ template "webui-fullname" . }}
+  annotations:
+    {{- range $key, $value := .Values.WebUi.LoadBalancerAnnotations }}
+      {{ $key }}: {{ $value | quote }}
+    {{- end }}
```

Comment:
Let's call them `ServiceAnnotations` and use this pattern:
```
{{- if .Values.WebUi.ServiceAnnotations }}
  annotations:
{{ toYaml .Values.WebUi.ServiceAnnotations | indent 4 }}
{{- end }}
```

Be sure to add `WebUi.ServiceAnnotations` to `values.yaml`, and set the default to empty braces (`{}`) because we're expecting a map.